spec/mspec/lib/mspec/helpers/ruby_exe.rb

```diff
@@ -144,6 +144,7 @@ def ruby_exe(code, opts = {})
 
       begin
         platform_is_not :opal do
+          puts "$ #{ruby_cmd(code, opts)}"
```

Comment:
This would print it on every `ruby_exe`, is that intended?
I think it's a useful debug aid, but maybe too noisy compared to printing the initial command line once at the start.